metaflow/plugins/kfp/README.md

```diff
@@ -4,14 +4,25 @@
 
 #### Steps to run Metaflow on Kubeflow: 
 
+#### Installing the correct forked version of Metaflow AND KFP
+Currently, to run Metaflow on our internal KFP, you need to install the correct versions of both.
+
+To install KFP:
+```
+    - git clone -b alexla/AIP-1676 https://github.com/alexlatchford/pipelines
+    - cd pipelines/sdk/python
+    - pip3 install -e .
+```
+To install Metaflow: `pip3 install --user --upgrade git+https://github.com/zillow/metaflow.git@bug/add_namespace_to_kfp_run`
```

Comment:
this would end up being the `feature/kfp` branch after merge?

Reply:
Good catch, resolved.